receiver/otelarrowreceiver/internal/arrow/arrow.go

```diff
@@ -553,6 +558,13 @@ func (r *receiverStream) recvOne(streamCtx context.Context, serverStream anyStre
 		} else if errors.Is(err, context.Canceled) {
 			return status.Error(codes.Canceled, "server stream shutdown")
 		}
+		if status, ok := status.FromError(err); ok {
```

Comment:
```suggestion

		if status, ok := status.FromError(err); ok {
```
Nit: Since both the above `if` and `else if` return, it may help to have an empty line here to show the separation of logic.

Reply:
Fixed.